panels/info/cc-info-panel.c

```diff
@@ -1469,7 +1471,12 @@ updates_apply_completed (GObject *object,
   if (error != NULL)
     {
       if (!g_error_matches (error, G_IO_ERROR, G_IO_ERROR_CANCELLED))
-        g_warning ("Failed to call Apply() on EOS updater: %s", error->message);
+        {
+          CcInfoPanel *self = user_data;
+          g_warning ("Failed to call Apply() on EOS updater: %s", error->message);
+          sync_state_from_updater (self, FALSE);
```

Comment:
Yeah, the commit message is a bit misleading -- that patch will effectively make it so when the daemon is already in an error state, and the call fails, such error state is propagated to the UI, undo-ing the initial state setup we do here: https://github.com/endlessm/gnome-control-center/blob/master/panels/info/cc-info-panel.c#L1581

I could write another follow-up patch that also considers the local error, regardless of what state the daemon is in, if you think it's worth it.